cmd/backup-manager/app/backup/backup.go

```diff
@@ -134,3 +135,128 @@ func constructOptions(backup *v1alpha1.Backup) ([]string, error) {
 	args = append(args, config.Options...)
 	return args, nil
 }
+
+// startlogBackupExec generates br args about log backup start and runs br binary to do the real backup work.
+func (bo *Options) startlogBackupExec(ctx context.Context, backup *v1alpha1.Backup) error {
```

Comment:
rename to startExecLogBackup

Reply:
If you want to make it different to the function in manager, you can add name it like this "doStartLogBackup".
Others naming can also follow the same pattern.

Reply:
ok